production/jaeger-production-template.yml

```diff
@@ -120,13 +120,13 @@ items:
       spec:
         containers:
         - name: jaeger-cassandra-schema
-          image: jaegertracing/jaeger-cassandra-schema:0.6
+          image: jaegertracing/jaeger-cassandra-schema:0.8
```

Comment:
if you are updating versions in this PR could you update also all-in-one?

Reply:
I reverted this change.